TESTING.md

```diff
@@ -111,4 +113,208 @@ Check file attributes of OV (ownership voucher) under aio-dir/stores/owner_vouch
+   │   ├── fdo-owner-onboarding-server.stdout.log
+   │   ├── fdo-rendezvous-server.stderr.log
+   │   ├── fdo-rendezvous-server.stdout.log
+   │   ├── fdo-serviceinfo-api-server.stderr.log
+   │   └── fdo-serviceinfo-api-server.stdout.log
+   ├── stores
+   │   ├── manufacturer_keys
+   │   ├── manufacturing_sessions
+   │   ├── mfg_sessions
+   │   ├── owner_onboarding_sessions
+   │   ├── owner_sessions
+   │   ├── owner_vouchers
+   │   ├── rendezvous_registered
+   │   ├── rendezvous_sessions
+   │   ├── serviceinfo_api_devices
+   │   └── serviceinfo_api_per_device
+   └── work
+   ```
+   
+   If you look in `aio-dir/configs` you'll have all the configuration files
+   needed by servers already configured with some defaults for you.
+   
+2. Modify `serviceinfo_api_server.yml`.
+
+   We'll modify that configuration file to specify what we want to do in the
+   onboarding process, in this case we will put a file in the Device to be
+   onbarded.
+   
+   Edit the `service_info:` `files:` section in `serviceinfo_api_server.yml` so
+   that it looks like this:
+   
+   ```yaml
+   files:
+   - path: /home/fedora/destination-dir/the-file-has-moved-here
+     permissions: 644
+     source_path: /home/fedora/source-dir/configuration-file.config
+   ```
+   
+   be mindful of the spaces and replace `/home/fedora/` in both paths with your
+   home directory.
+   
+   You can look at [examples/config/service-info-api-server.yml](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/examples/config/serviceinfo-api-server.yml)
+   for a more detailed example.
+
+3. Create the required directories and file.
+
+   ```bash
+   mkdir ~/destination-dir
+   mkdir ~/source-dir
+   echo "some configuration" > ~/source-dir/configuration-file.config
+   ```
+   
+   In this demo `~/destination-dir` represents a directory in the Device, but
+   for the sake of this demo it lives in your current machine. Similarly
+   `~/source-dir/configuration-file.config` is a file that would exist in the
+   `serviceinfo-api` server in production, but for demo purposes, this file is
+   also located in your system.
+   
+3. Device set-up
+   
+   Your system will also represent the Device that we will onboard during this
+   demo, so we need to set it up. This process represents what would be done
+   during the manufacturing supply line.
+   
+   Create the `rendezvous_info.yml` file.
+   
+   Copy the `rendezvoys_info:` section of the
+   `aio-dir/confi/manufacturing_server.yml` into a file named
+   `rendezvous_info.yml` and modify it so it has this same structure:
+   
+   ```yaml
+   ---
+   - deviceport: 8082
+     ip_address: 192.168.122.44
+     ownerport: 8082
+     protocol: http
+   ```
+   be mindful of the spaces. Your `ip_address` and `deviceport` may change but
+   the structure should follow the given example.
+   
+   Generate an OV and a `device_credential`.
+   
+   ```bash
+   fdo-owner-tool initialize-device 54321 ov device_credential \
```

Comment:
What is the `54321` value used here?

Reply:
Nice catch, added some clarifications below